core-bundle/src/Resources/contao/templates/frontend/fe_page.html5

```diff
@@ -21,7 +21,7 @@
   <?php $this->endblock(); ?>
 
 </head>
-<body id="top"<?php if ($this->class): ?> class="<?= $this->class ?>"<?php endif; ?><?php if ($this->onload): ?> onload="<?= $this->onload ?>"<?php endif; ?>>
+<body id="top"<?php if ($this->class): ?> class="<?= $this->class ?>"<?php endif; ?><?php if ($this->onload): ?> onload="<?= $this->onload ?>"<?php endif; ?> itemscope itemtype="https://schema.org/WebPage">
```

Comment:
@Toflar It seems that you had removed this in #2962. Was this on purpose? Because the new JSON-LD graph does not contain a `WebPage` item.

Reply:
Ah right, fixed in https://github.com/contao/contao/pull/3107.